PowerPointLabs/PowerPointLabs/PictureSlidesLab/Model/StyleOptions.cs

```diff
@@ -481,6 +485,42 @@ public bool IsInsertReference
             }
         }
 
+        private int _fontSize;
+
+        public int FontSize
+        {
+            get { return _fontSize; }
+            set
+            {
+                _fontSize = value;
+                OnPropertyChanged("FontSize");
+            }
+        }
+
+        private Alignment _imageRefAlignment;
```

Comment:
use int here, and provide a getter function to map this int to Alignment, similar to GetTextBoxAlignment()